suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -410,3 +411,28 @@ tests:
           desc: RBD Persistent Cache invalidate image level
           module: test_rbd_persistent_writeback_cache_invalidate.py
           name: PWL cache path validation at image level
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: client                        # PWL at client
+        cache_file_size: 1073741824          # 1 GB
+        rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+        client: node5
```

Comment:
as we face some issue with passing `client` node explicitly, it's good to get that node through module code.

Reply:
Since persistent cache tests are running only in baremetal mero config, its fine to retain it here for now. We can change it when we are doing SDK implementation for persistent cache if required. Since this is unnecessary here for this test case. 
In your test case, dm cache since it was running on both cephci and baremetal we had to do the change.